### ISSUE TITLE
XCode 10 GM and Swift 4.2 fail to build

### ISSUE THREAD

**Issue by jlaws** [HHK1/PryntTrimmerView#34]
```
/Users/joelaws/Workspace/kratos/Pods/PryntTrimmerView/PryntTrimmerView/Classes/Cropper/CropMaskView.swift:33:30: 'kCAFillRuleEvenOdd' has been renamed to 'CAShapeLayerFillRule.evenOdd'
/Users/joelaws/Workspace/kratos/Pods/QuartzCore.kCAFillRuleEvenOdd:3:12: 'kCAFillRuleEvenOdd' was obsoleted in Swift 3
/Users/joelaws/Workspace/kratos/Pods/PryntTrimmerView/PryntTrimmerView/Classes/Cropper/CropMaskView.swift:108:64: 'kCAMediaTimingFunctionEaseInEaseOut' has been renamed to 'CAMediaTimingFunctionName.easeInEaseOut'
/Users/joelaws/Workspace/kratos/Pods/QuartzCore.kCAMediaTimingFunctionEaseInEaseOut:3:12: 'kCAMediaTimingFunctionEaseInEaseOut' was obsoleted in Swift 3
/Users/joelaws/Workspace/kratos/Pods/PryntTrimmerView/PryntTrimmerView/Classes/Cropper/CropMaskView.swift:109:30: 'kCAFillModeBoth' has been renamed to 'CAMediaTimingFillMode.both'
/Users/joelaws/Workspace/kratos/Pods/QuartzCore.kCAFillModeBoth:3:12: 'kCAFillModeBoth' was obsoleted in Swift 3
/Users/joelaws/Workspace/kratos/Pods/PryntTrimmerView/PryntTrimmerView/Classes/ThumbSelectorView.swift:122:50: 'kCMTimeZero' has been renamed to 'CMTime.zero'
/Users/joelaws/Workspace/kratos/Pods/CoreMedia.kCMTimeZero:3:12: 'kCMTimeZero' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/PryntTrimmerView/PryntTrimmerView/Classes/ThumbSelectorView.swift:123:51: 'kCMTimeZero' has been renamed to 'CMTime.zero'
/Users/joelaws/Workspace/kratos/Pods/CoreMedia.kCMTimeZero:3:12: 'kCMTimeZero' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/PryntTrimmerView/PryntTrimmerView/Classes/Parents/AssetVideoScrollView.swift:139:65: 'AVAssetImageGeneratorResult' has been renamed to 'AVAssetImageGenerator.Result'
/Users/joelaws/Workspace/kratos/Pods/AVFoundation.AVAssetImageGeneratorResult:2:18: 'AVAssetImageGeneratorResult' was obsoleted in Swift 4.2
/Users/joelaws/Workspace/kratos/Pods/PryntTrimmerView/PryntTrimmerView/Classes/Parents/AssetVideoScrollView.swift:162:78: 'UIImageOrientation' has been renamed to 'UIImage.Orientation'
/Users/joelaws/Workspace/kratos/Pods/UIKit.UIImageOrientation:2:18: 'UIImageOrientation' was obsoleted in Swift 4.2
```

**Comment by HHK1** [comment]
@jlaws Can you try #35 and tell me if everything works smoothly ?

**Comment by HHK1** [comment]
I've published a pre-release here: https://github.com/HHK1/PryntTrimmerView/releases/tag/3.0.0-beta.1

**Comment by jlaws** [comment]
@HHK1 can we get this merged into master and have the main pod work for Swift 4.2?

**Comment by HHK1** [comment]
@jlaws Already merged into master, and tagged under 3.0.0. 
However I'm waiting to regain access to the trunk to publish it on cocoapods
